xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-ui/src/main/resources/Main/SolrSearchMacros.xml

```diff
@@ -67,7 +67,10 @@
     &lt;/div&gt;
     &lt;div class="col-xs-12 col-sm-6"&gt;
       &lt;div class="input-group"&gt;
-        &lt;input type="search" name="text" class="form-control withTip useTitleAsTip"
+        &lt;label class="sr-only" for='search-page-bar-input'&gt;
```

Comment:
Minor point, but it seems inconsistent to use `'` and `"` for different attributes of the same element.

Reply:
Yeah, I should have used single quotes for the class. I think in some cases my editor autocompletes attributes with double quotes, I need to see how to change this :)
Addressed in https://github.com/xwiki/xwiki-platform/pull/2187/commits/63d5756b9f44d528d90c23acc755f423acceb08a